### PR TITLE
fix sandbox scripts for kfile changes

### PR DESCRIPTION
**Pull request by ctb**
ref #849, this PR fixes a few sandbox scripts that are (a) run in khmer-recipes and (b) use the khmer.kfile functionality.

### CODE REVIEW

**Comment by ctb** [comment]
- [ ] Is it mergable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage.
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by ctb** [comment]
Ready for merge. Note: pre-existing sandbox scripts only; be gentle.

**Comment by mr-c** [comment]
LGTM

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,8 @@
+2015-02-25  Titus Brown  <titus@idyll.org>
```
Comment (mr-c) [inline ChangeLog]: Update date to be same day or newer than the entry below.

Comment (ctb) [inline ChangeLog]: Fixed, should be ready for merge.

**Comment by mr-c** [comment]
Done!

**Pull request merged**